README.md

```diff
@@ -27,6 +27,25 @@ NOTE: When we build `techdocs-cli` it copies the output `embedded-techdocs-app`
 bundle into the `packages/techdocs-cli/dist` which is then published with the
 `@techdocs/cli` npm package.
 
+If you're changing files inside `@backstage/techdocs-common` locally and want to run it locally with `techdocs-cli` you can run:
+
+```sh
+cd backstage/packages/techdocs-common`
```

Comment:
Maybe explicitly mention that this is a separate repository? https://github.com/backstage/backstage

Reply:
@OrkoHunter done! I added a note about it below the command, let me know if you had any other suggestion 😄